src/accountsdb/genesis_config.zig

```diff
@@ -83,11 +83,11 @@ pub const Rent = extern struct {
     /// distributed to validators.
     burn_percent: u8,
 
-    pub fn random(rand: std.Random) Rent {
+    pub fn initRandom(random: std.Random) Rent {
         return .{
-            .lamports_per_byte_year = rand.int(u64),
-            .exemption_threshold = @bitCast(rand.int(u64)),
-            .burn_percent = rand.uintAtMost(u8, 100),
+            .lamports_per_byte_year = random.int(u64),
+            .exemption_threshold = @bitCast(random.int(u64)),
```

Comment:
You've already done a bit of cleanup in this PR which I really appreciate, so if you don't mind could you change these bit casts to call `random.float(f64)`? ditto the ones in the `Inflation` struct.

Reply:
done b4ffcdef27060b8d7bca2cf85e36c714f9db647c